frontend/packages/client/src/contexts/NotificationService.js

```diff
@@ -28,6 +34,16 @@ const updateCommunitySubscriptionState = (
   return newCommunitySubscription;
 };
 
+const IS_LOCAL_DEV = process.env.REACT_APP_APP_ENV === 'development';
+
+if (IS_LOCAL_DEV) {
+  Leanplum.setAppIdForDevelopmentMode(LEANPLUM_APP_ID, LEANPLUM_DEV_KEY);
+} else {
+  Leanplum.setAppIdForProductionMode(LEANPLUM_APP_ID, LEANPLUM_PROD_KEY);
+}
+
+Leanplum.start();
```

Comment:
not saying this is the case, but i've had issues with other API libraries where calling `.start()` from the top level can cause the app to crash/error if that underlying api service is down,. Generally I think it's better to start the service only when you need it (when a user triggers a part of the app that explicitly needs leanplum to render/work properly)

Reply:
We need to run this once when the app starts, we can potentially move it to the login event, can you suggest a file/location to move this code to?

Reply:
@tuffon having it called on login makes sense to me since we wont be able to get a user's preferences without that anyway 👍

Reply:
I'm not familiar with where that logic should be placed can you suggest a location? otherwise I will keep it here

Reply:
Can we wrap this section of code up into a `startLeanplum` and put it into `api/notificationService.js`? So the implementation details of leanPlum can be encapsulated.

Reply:
sure, when you integrate you can put the function call where you need it